### PR TITLE
bugtool: improve lb dump support and add monitor helper for k8s tests

### LINKED ISSUE #8081: bugtool: dump everything from services_v2 table
Especially slave mappings, so we can correlate them to backend map. Maybe a `cilium bpf lb raw` command would be nice, where we both dump services+backend map.

### PR DESCRIPTION
**Pull request by borkmann**
Fixes: #8081

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/8151)

### CODE REVIEW

**Comment by borkmann** [comment]
test-me-please

File: bugtool/cmd/configuration.go [hunk 1]
```diff
@@ -59,6 +59,13 @@ func defaultCommands(confDir string, cmdDir string, k8sPods []string) []string {
 		"dmesg --time-format=iso",
 		"bpftool map show",
 		"bpftool prog show",
+		// LB and CT map for debugging services; using bpftool for a reliable dump
```
Comment (brb) [inline bugtool/cmd/configuration.go:68]: Maybe we should add dumping for IPv6?

Comment (borkmann) [reply bugtool/cmd/configuration.go:68]: Sigh, agree. Done now.

**Comment by borkmann** [comment]
test-me-please

**Review by brb — APPROVED** [review]

**Pull request merged**